core/src/main/java/com/novoda/merlin/NetworkStatus.java

```diff
@@ -2,7 +2,7 @@
 
 public class NetworkStatus {
 
-    public enum State {
+    private enum State {
```

Comment:
These two states are exposed through the `isAvailable` call, no need to make this publically available.